microservices/nris_api/backend/app/nris/resources/document.py

```diff
@@ -0,0 +1,34 @@
+from flask import request
+from flask_restplus import Resource
+
+from werkzeug.exceptions import BadRequest, NotFound, InternalServerError
+
+from app.extensions import api, db
+from app.nris.utils.access_decorators import requires_role_nris_view
+
+from app.nris.models.document import Document as Model, DOCUMENT_RESPONSE_MODEL as RESPONSE_MODEL
+
+module_path = 'documents'
```

Comment:
I like the new use of module_path 💯 Saves the init file overhead and repetition on routes

Reply:
@Jsyro 🥇